api/app/routers/hfi_calc.py

```diff
@@ -352,7 +357,7 @@ async def get_fire_centres(response: Response):
         logger.info('/hfi-calc/fire-centres')
         # we can safely cache the fire centres, as they don't change them very often.
         # the eco-division logic is very slow, and chomps up 2 seconds!
-        response.headers["Cache-Control"] = "max-age=86400"
+        response.headers["Cache-Control"] = "max-age=0"
```

Comment:
We need to have some sort of caching here - that eco-division thing really messes up the load. If we're going to lose browser caching, then I think we need to switch to using redis to cache the response. We can then invalidate the cache entry when we do a post that changes the fire centres.

Reply:
This change to the age of the cache contradicts the comment immediately above it. Please update.